src/test/java/seedu/address/logic/commands/DeleteSessionCommandTest.java

```diff
@@ -24,25 +30,50 @@
 
     private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
 
-    /*
+
     @Test
-    public void execute_validNameValidIndex_success() {
-        Student studentWithSession = model.getAddressBook().getStudentList().get(INDEX_FIRST_STUDENT.getZeroBased());
+    public void execute_validNameValidIndex_success() throws CommandException {
+        AddressBook deletionAddressBook = new AddressBook();
```

Comment:
we put this outside the method, at the top of the class?
to use in every other method for test

Reply:
Are we adding more tests to DeleteSessionCommandTest?

Reply:
Yea I think like failure cases and all that

Reply:
Improves coverage to do that right i hope

Reply:
hollup if we abstract this method and put it at the top of the class, then won't we be using the same edited addressbook for other test cases as well?

Reply:
oh right! so i guess it has to stay as it is rn